doc/output-details-and-examples/src/data-sets/simple-list-data-sets.tex

```diff
@@ -66,6 +66,31 @@ \subsection{Chillers.idf}\label{chillers.idf}
   ! ElectricEIRChiller Carrier 23XL 724kW/6.04COP/Vanes  Centrifugal   724 (206)   6.04   Inlet Vanes
 \end{lstlisting}
 
+\subsection{CodeCompliantEquipment.idf}\label{codecompliantequipment.idf}
+
+This dataset includes performance curve objects that describe the part load performance of building energy code minimally compliant equipment. The dataset provides curves for the following EnergyPlus objects: Chiller:Electric:EIR. Sets of curves are provided for the following building energy codes minimum requirements: ASHRAE Standard 90.1-2019 Table 6.8.1-3, ASHRAE Standard 90.1-2019 Table G3.5.3.
+
+Generating performances curves meeting both full and part load efficiency numbers require solving a largely underdetermined system of equations. The number of unknowns depend on the type of curves, which mostly depend on the algorithm used by the software and used in models. It also means that an infinite number of solutions exists. One way to solve this issue is to turn it into a constrained problem and use a simple genetic algorithm (GA) with a well-defined objective or cost function to determine potential solutions with physical significance. A GA can have a short computational time, be scalable, versatile, find solutions to problems with large number of parameters to problems with multiple local optima so it is a good candidate for this task. The algorithm used to generate the curves in this dataset starts with a ``seed'' set of curves that can be user-modifiable, target full load, and part load efficiency as inputs.
```

Comment:
Would it be possible to describe some specifics for how the cost function was defined?

Reply:
@lymereJ can you respond to @EnergyArchmage here in this PR and then sometime after release add an entry to the documentation or dataset header or something that also captures this?

Reply:
@EnergyArchmage @Myoldmopar -

Sorry for the belated response. Here's the fitness function that we used, presented in pseudo-code:

`fitness = (iplv_score + full_load_score + normalization_score + 0.5 * rsme_score) / 3.5`

Where:
- `iplv_score` is the absolute difference between the calculated IPLV and the target
- `full_load_score` is the absolute difference between the calculated full load efficiency and the target
- `normalization_score` is a calculated value that expresses how close to 1 each curve are at rated conditions
- `rsme_score` RSME between the curves and "seed" curves for a range of data points, the idea here is to encourage curves being "closer" to the seed curves

Note: all score are arbitrarily weighted, all of them are equally weighted except the `rsme_score`.

Reply:
Thanks for this,  but what is the "target"?  MFG reported test results for the IPLV points?  I can wait for the paper.  Anyway this is great stuff.